pai-management/k8sPaiLibrary/template/kubelet.sh.template

```diff
@@ -48,4 +48,5 @@ docker run \
   --logtostderr=true \
   --pod-infra-container-image {{ clusterconfig['dockerregistry'] }}/pause-amd64:3.0 \
   --eviction-hard="memory.available<5%,nodefs.available<5%,imagefs.available<5%,nodefs.inodesFree<5%,imagefs.inodesFree<5%" \
+  --image-pull-progress-deadline=6m \
```

Comment:
6m is enough? Maybe, this time should be longer?

Reply:
It's enough for now.
We may add an configuration entry later, maybe in another PR.

Reply:
Let's set it 10 min?

Reply:
Cool, I will update.

Reply:
If no pulling progress is made before this deadline, the image pulling will be cancelled. What will happen if the pulling canceled?

Reply:
@YitongFeng , it will retry.
If lucky, next retry may succeed, or it won't success forever.